duckduckgo/duckduckgo-autofill
# Fix handler regression

**Reviewer:** @shakyShane 
**Asana:** https://app.asana.com/0/1200930669568058/1205581668483223/f

## Description
We [recently](https://github.com/duckduckgo/duckduckgo-autofill/pull/333) introduced a regression on extensions where we were no longer protecting against self-closing modals, meaning trying to autofill would also close the modal in certain cases. This fixes the regression and the tests that didn't catch it 🙃. I've also slipped in a minor accuracy update.

## Steps to test
Tests now actually work. To see this in action, you can try this in https://privacy-test-pages.site/autofill/autoprompt/2-form-in-modal.html using the extension. The production extension will close the modal, but this version won't.
